simpledotorg/simple-server
# Add  `patient initiated on treatment` card at the end of monthly reports

**Story card:** [sc-8978](https://app.shortcut.com/simpledotorg/story/8978)

## Because

The yellow card saying "Where are the patients initiated on treatment data?" was missing on the monthly reports page

## This addresses

Adds a yellow card saying "Where are the patients initiated on treatment data?" on the monthly reports page

## Test instructions

- Checkout to branch
- Enable `new_progress_tab_v1` feature flag
- Sidebar>Reports>Choose any facility>Progress Tab> Monthly 
- Scroll down to the end. You should be able to see the card
